pom.xml

```diff
@@ -204,6 +218,20 @@
       <artifactId>passay</artifactId>
       <version>1.6.0</version>
     </dependency>
+    <dependency>
+      <groupId>org.elasticsearch.client</groupId>
+      <artifactId>elasticsearch-rest-high-level-client</artifactId>
+      <version>${elastic-search-version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.elasticsearch</groupId>
+      <artifactId>elasticsearch</artifactId>
+      <version>${elastic-search-version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
```

Comment:
what's this dependency being used for?

Reply:
It fixed an error I was getting a few days ago regarding logging, but I can neither remember exactly what it was nor can I reproduce it now as things work fine without it, I'll remove the dependency.